widgetssdk/src/main/java/com/glia/widgets/config/component/base/Color.java

```diff
@@ -0,0 +1,13 @@
+package com.glia.widgets.config.component.base;
+
+import androidx.annotation.ColorInt;
+
+/**
+ * Represents ARGB color fields from remote config e.g. #FF44DD55
+ * with the help of {@link com.glia.widgets.config.deserializer.ColorDeserializer} it guarantees
+ * that {@link #getColor()} will return parsed valid color value
+ */
+public interface Color {
```

Comment:
What is the purpose of interfaces like Color?
If it's used only for `ColorImpl` (public) and there don't seem to be new classes that will implement it, I would suggest simplifying the architecture and using classes without interfaces.

But if you really see a reasonable need to introduce interfaces - let's keep them, of course.